docs/README.md

```diff
@@ -670,6 +670,15 @@ $data = [
 ];
 ```
 
+Or send reusable Emails via the Mailer object:
+```php
+$data = [
+    'settings' => $mailerObject,
+    'action' => 'myReusableEmail',
+    'actionVars' => [$var1, $var2, $var3]
```

Comment:
The code further down uses

    if (!empty($data['vars'])) {

for settings as array instead of object.

I wonder if we should keep this more consistent by using the same key
```php
$data = [
    'settings' => $mailerObject,
    'action' => 'myReusableEmail', // instead of content or headers
    'vars' => [$var1, $var2, $var3] // same
```

Reply:
alright, I will do that